community-operators/federatorai/federatorai.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+          verbs:
+          - get
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - get
+          - create
+          - update
+        - apiGroups:
+          - ''
+          resources:
+          - secrets
+          verbs:
+          - create
+          - watch
+          - list
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - validatingwebhookconfigurations
+          verbs:
+          - watch
+          - create
+          - update
+          - list
+        - apiGroups:
+          - ''
+          resources:
+          - services
+          verbs:
+          - watch
+          - create
+          - update
+          - list
+      - serviceAccountName: alameda-datahub
+        rules:
+        - apiGroups:
+          - ''
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - autoscaling.containers.ai
+          resources:
+          - alamedarecommendations
+          verbs:
+          - get
+          - update
+      - serviceAccountName: alameda-evictioner
+        rules:
+        - apiGroups:
+          - ''
+          resources:
+          - pods
+          verbs:
+          - get
+          - delete
+      - serviceAccountName: admission-controller
+        rules:
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - mutatingwebhookconfigurations
+          verbs:
+          - get
+          - create
+          - update
+        - apiGroups:
+          - ''
+          resources:
+          - pods
+          verbs:
+          - get
+          - list
+          - patch
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+          - list
+        - apiGroups:
+          - ''
+          resources:
+          - replicationcontrollers
+          verbs:
+          - get
+          - list
+        - apiGroups:
+          - apps.openshift.io
+          resources:
+          - deploymentconfigs
+          verbs:
+          - get
+          - list
+      - serviceAccountName: alameda-ai
+        rules: []
+      deployments:
+      - name: federatorai-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: federatorai-operator
+            strategy:
+              type: Recreate
+          template:
+            metadata:
+              labels:
+                name: federatorai-operator
+            spec:
+              serviceAccountName: federatorai-operator
+              containers:
+              - command:
+                - federatorai-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Please set the value of the `WATCH_NAMESPACE` env variable to the following:
```
valueFrom:
  fieldRef:
    fieldPath: metadata.annotations['olm.targetNamespaces']
```

Reply:
Yes got it. Thanks. We had pushed a new commit to fix it.